plinth/modules/deluge/__init__.py

```diff
@@ -38,7 +38,7 @@
 
 managed_packages = ['deluged', 'deluge-web']
 
-title = _('BitTorrent Web Client (Deluge)')
+title = ('BitTorrent Web Client \n (Deluge)')
```

Comment:
The _ in front of the string is important. It's an alias for ugettext_lazy, which we need for translations to work.